apps/api/src/server/app/case-officer/case-officer.confirm.test.js

```diff
@@ -6,9 +6,19 @@ import { app } from '../../app.js';
 const request = supertest(app);
 
```

Comment:
Could we please add a test here to check that a 400 is throws when
- The appeal is not in a state ready to receive confirmation of the LPA questionnaire
- Some unexpected body attributes are provided
- When request is missing some information in the body, such as when we mark `applicationPlansToReachDecisionMissingOrIncorrect` but do not provide a matching description